### PR TITLE
Update roadmap 2022

### PR DESCRIPTION
**Pull request by mapedraza**
This PR updates the roadmap from January 2020 to March 2022

### CODE REVIEW

**Comment by mapedraza** [comment]
@fgalan this PR is ready to merge

File: docs/roadmap.md [hunk 1]
```diff
@@ -24,27 +24,17 @@ Disclaimer:
...
 The following list of features are planned to be addressed in the medium term, typically within the subsequent
 release(s) generated in the next **9 months** after next planned release:
 
--   Improve template processing, extending it to StructuredValue/JSON attributes.
--   Support for dealing with several Context Brokers from the same Perseo instance. To do so, update actions will
-    support to configure target context broker, if not provided such configuration, the default context broker will be
-    used.
 -   Rule scope beyond subservice: support dynamic creation of EPL context associated to notification path
+-   Accept scripts and java snipets
+-   No signal rules based in NGSIv2 API instead of MongoDB direct access #549
```
Comment (mapedraza) [inline docs/roadmap.md:37]: PR https://github.com/telefonicaid/perseo-fe/pull/695 is solving this issue?

Comment (fgalan) [reply docs/roadmap.md:37]: No. Issue #549 is still pending.

Comment (mapedraza) [reply docs/roadmap.md:37]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**